FBSnapshotTestCase/FBSnapshotTestCase.modulemap

```diff
@@ -0,0 +1,6 @@
+framework module FBSnapshotTestCase {
+    umbrella header "FBSnapshotTestCase.h"
+
+    export *
+    module * { export * }
```

Comment:
TBH I copied this module map from the generated one from CocoaPods. Also @segiddens [pointed out](https://twitter.com/segiddins/status/606574460603375616) I should add a module map.